<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion__reservation</class>
 <widget class="QMainWindow" name="gestion__reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1513</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pushButton_ViewLogs">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-40</y>
      <width>1611</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 155, 211);
background-color: rgb(237, 237, 237);</string>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QLineEdit" name="lineEdit_Prix">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>381</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>660</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>660</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>660</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>611</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_T">
      <property name="geometry">
       <rect>
        <x>762</x>
        <y>73</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>523</x>
        <y>84</y>
        <width>182</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}
</string>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>450</y>
        <width>161</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGraphicsView {
    background-color: #f8f8f8; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #f8f8f8; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Stats">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>407</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>statistique</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_DebutSejour">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>221</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_FinSejour">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>271</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_DateRes">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>321</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Chambre">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>431</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <item>
       <property name="text">
        <string>chambre simple</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>chambre double</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>suite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>chambre avec balcon</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonSend">
      <property name="geometry">
       <rect>
        <x>845</x>
        <y>73</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>arduino</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonExportPDF">
      <property name="geometry">
       <rect>
        <x>762</x>
        <y>105</y>
        <width>81</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ViewLogs_2">
      <property name="geometry">
       <rect>
        <x>845</x>
        <y>105</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>historique </string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>371</width>
        <height>218</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calendar">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>calendrier</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_Chambres">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>540</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_Chambres">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: #ffffff; /* Couleur d'arrière-plan (blanc) */
    color: black;             /* Couleur du texte (noir) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="menu">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>500</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #004aad; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #2980b9; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
      <property name="text">
       <string>menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>1001</width>
        <height>741</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Users/i__t__s/Pictures/maram.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>400</y>
        <width>113</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #ffffff; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>400</y>
        <width>90</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff; /* Couleur d'arrière-plan (bleu clair dans cet exemple) */
    color: white;             /* Couleur du texte (blanc dans cet exemple) */
    border: 1px solid #ffffff; /* Bordure avec une teinte légèrement plus foncée */
    border-radius: 5px;       /* Coins arrondis (facultatif) */
}</string>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>lineEdit_Prix</zorder>
     <zorder>lineEdit_ID</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>tableView</zorder>
     <zorder>pushButton_T</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>graphicsView</zorder>
     <zorder>pushButton_Stats</zorder>
     <zorder>dateEdit_DebutSejour</zorder>
     <zorder>dateEdit_FinSejour</zorder>
     <zorder>dateEdit_DateRes</zorder>
     <zorder>comboBox_Chambre</zorder>
     <zorder>pushButtonSend</zorder>
     <zorder>pushButtonExportPDF</zorder>
     <zorder>pushButton_ViewLogs_2</zorder>
     <zorder>calendarWidget</zorder>
     <zorder>pushButton_calendar</zorder>
     <zorder>frame_Chambres</zorder>
     <zorder>menu</zorder>
     <zorder>lineEdit</zorder>
     <zorder>lineEdit_2</zorder>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1513</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
